lib/xenops_server.ml

```diff
@@ -84,6 +84,7 @@ type atomic =
   | VM_remove of Vm.id
   | PCI_plug of Pci.id
   | PCI_unplug of Pci.id
+  | VGPU_set_active of Vgpu.id * bool
```

Comment:
Let's move this down just above `VGPU_start`, just to group similar things together.

Reply:
Yes.  Thanks.